Reviewer/src/main/java/com/softserveinc/reviewer/annotation/Collection.java

```diff
@@ -0,0 +1,14 @@
+package com.softserveinc.reviewer.annotation;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import com.google.inject.BindingAnnotation;
+
+@Retention(RUNTIME)
+@Target({ ElementType.FIELD, ElementType.PARAMETER })
+@BindingAnnotation
+public @interface Collection {}
```

Comment:
You should avoid overlapping with standard java classes. Collections, Lists and so on are not the best names for internal usage. That would make it hard to use both in one place.